README.md

```diff
@@ -40,6 +40,7 @@ Project is composed of multiple Maven sub-modules, each corresponding to a jar:
 * [jr-retrofit2](../../tree/master/jr-retrofit2) contains `jackson-jr` - based handlers for [Retrofit 2](https://square.github.io/retrofit/) library
     * Depends on `jackson-jr` and `Retrofit` API jars, and indirectly on `jackson-core`
 * [jr-annotation-support](../../tree/master/jr-annotation-support) contains extension with support for a subset of core [Jackson annotations](../../../jackson-annotations)
+* [jr-extension-datetime](../../tree/master/jr-extension-datetime) contains extension with support for `LocalDateTime`
```

Comment:
Although it is just one type for now, could mention it is for supporting "a subset of Java 8 Date/Time types", here, and then in README.md for extension list `DateTime`.